azure-pipelines.yml

```diff
@@ -31,28 +31,49 @@ variables:
   shouldPublish: $[or(eq(variables['Build.SourceBranch'], 'refs/heads/dev'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release/'))]
   ${{ if startsWith(variables['Build.SourceBranch'], 'refs/heads/') }}:
     branchName: $[ replace(variables['Build.SourceBranch'], 'refs/heads/', '') ]
+    branchToCompare: $[ variables['Build.SourceBranch'] ]
```

Comment:
Do we need additional manipulation of 'Build.SourceBranch' here?

Reply:
made steps more straightforward, without --since cause that added a lot of what-ifs and branching